internal/tiltfile/skylarkTypes.go

```diff
@@ -204,17 +205,18 @@ func addMount(thread *skylark.Thread, fn *skylark.Builtin, args skylark.Tuple, k
 		return nil, err
 	}
 
-	var lp localPath
+	m := mount{}
 	switch p := src.(type) {
 	case localPath:
-		lp = p
+		m.src = p
```

Comment:
i thought localPaths had git repos attached?

Reply:
A mount can either be from a gitRepo, or from a plain old path on disk. localPAths are those plain old paths, and they won't have a gitRepo.